### PR TITLE
OLS-440: Read password for Postgres from file

### PR DESCRIPTION
**Pull request by tisnik**
## Description

Read password for Postgres from file

## Type of change

- [ ] Refactor
- [x] New feature
- [ ] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Configuration Update
- [ ] Bump-up dependent library

## Related Tickets & Documents

- Related Issue #[OLS-440](https://issues.redhat.com//browse/OLS-440)

### CODE REVIEW

File: ols/app/models/config.py [hunk 1]
```diff
@@ -311,11 +311,20 @@ class PostgresConfig(BaseModel):
     port: PositiveInt = constants.POSTGRES_CACHE_PORT
     dbname: str = constants.POSTGRES_CACHE_DBNAME
     user: str = constants.POSTGRES_CACHE_USER
+    password_path: Optional[str] = None
     password: Optional[str] = None
     require_ssl: bool = False
     ca_cert_path: Optional[str] = None
     max_entries: PositiveInt = constants.POSTGRES_CACHE_MAX_ENTRIES
 
+    def __init__(self, **data: Any) -> None:
```
Comment (onmete) [inline ols/app/models/config.py:320]: Is the init the right place to do that? Isn't there some coercion/post init available in pydantic to do that?

Comment (tisnik) [reply ols/app/models/config.py:320]: I looked at examples and every of them do it in __init__

Comment (tisnik) [reply ols/app/models/config.py:320]: basically init is called when all attributes are filled in already

Comment (onmete) [reply ols/app/models/config.py:320]: ok then

**Pull request merged**